doc/classes/XRPositionalTracker.xml

```diff
@@ -96,14 +96,12 @@
 			<param index="0" name="name" type="String" />
 			<param index="1" name="vector" type="Vector2" />
 			<description>
-				Emitted when a thumbstick or thumbpad on this tracker moves.
```

Comment:
You lost descriptions here.

Reply:
thanks, it was giving me a hard time with that one because it flipped the order.

Reply:
dangit, for some reason it kept changing it back...